components/core/GithubCodeLink.js

```diff
@@ -0,0 +1,29 @@
+// icons
+import { FiGithub } from 'react-icons/fi';
+
+const GithubCodeLink = ({ url }) => {
+  return (
+    <div className="flex-colums">
+      <a
+        target="_blank"
+        rel="noreferrer"
+        href={url}
+        style={{
+          margin: '50px',
```

Comment:
don't use an inline style like this. Use the tailwind syntax for specifying margin.